### PR TITLE
Added exception handling for force rotate logs

### PR DESCRIPTION
**Pull request by OleksandrKozodoi**
Signed-off-by: Oleksandr Kozodoi <oleksandrx.kozodoi@intel.com>

### Description of PR

Summary:
PR #2161 added force rotate logs. We are observing that log rotate contains duplicate log entries (see below). In this case `duthost.shell("/usr/sbin/logrotate -f /etc/logrotate.conf > /dev/null 2>&1") ` command is failed and logs are rotated. As result test case is failed as well. This commit is to address this issue.
```
sudo /usr/sbin/logrotate -d /etc/logrotate.conf

reading config file /etc/logrotate.conf
including /etc/logrotate.d
reading config file alternatives
reading config file apt
reading config file btmp
reading config file dpkg
error: dpkg:10 duplicate log entry for /var/log/alternatives.log
error: found error in file dpkg, skipping
reading config file monit

```
```
cat /etc/logrotate.d/alternatives

/var/log/alternatives.log {
        monthly
        rotate 12
        compress
        delaycompress
        missingok
        notifempty
        create 644 root root
}
cat /etc/logrotate.d/dpkg

/var/log/dpkg.log {
        size 100k
        rotate 1
        compress
        delaycompress
        missingok
        notifempty
        create 644 root root
}
/var/log/alternatives.log {
        size 100k
        rotate 1
        compress
        delaycompress
        missingok
        notifempty
        create 644 root root
}
```
### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Loganalyzer failed if we rotate logs and duplicate log entries are present in configuration files.
#### How did you do it?
Added exception handling for logrotate command and log appropriate warning message.
#### How did you verify/test it?
Run test.

### CODE REVIEW

**Comment by OleksandrKozodoi** [comment]
@lolyu Please review

**Comment by OleksandrKozodoi** [comment]
@akokhan @SavchukRomanLv Please review

File: tests/common/plugins/loganalyzer/__init__.py [hunk 1]
```diff
@@ -15,10 +16,15 @@ def loganalyzer(duthost, request):
...
     # Force rotate logs
-    duthost.shell(
-        "/usr/sbin/logrotate -f /etc/logrotate.conf > /dev/null 2>&1"
-        )
+    try:
+        duthost.shell(
+            "/usr/sbin/logrotate -f /etc/logrotate.conf > /dev/null 2>&1"
+            )
+    except RunAnsibleModuleFail as e:
+        logging.warning("logrotate: duplicate log entry in configuration files")
```
Comment (akokhan) [inline tests/common/plugins/loganalyzer/__init__.py:26]: I think the reason of failure can be different. I'd just log exception message.

**Review by akokhan — APPROVED** [review]

**Comment by lolyu** [comment]
I have two questions:
1. which test case caused this symptom?
2. Should we remove the redundant log entry?

File: tests/common/plugins/loganalyzer/__init__.py (hunk 1 above)
Comment (OleksandrKozodoi) [reply tests/common/plugins/loganalyzer/__init__.py]: Done.

**Comment by OleksandrKozodoi** [comment]
> I have two questions:
> 
> 1. which test case caused this symptom?
> 2. Should we remove the redundant log entry?

1. All test cases
2. We could remove the redundant log entry, but other packages could reproduce this issue in the future. This issue blocks testing the main functionality of SONiC so we should suppress it. The Dev team is working on the issue with the redundant log entry.

**Review by lolyu — APPROVED** [review]
LGTM

**Pull request merged**